radix_sort/RadixSort.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * This Java program implements the Radix sort algorithm
+ * It's a non-comparative based sorting algorithm, hence its worse case time
+ * complexity is O(kn), and space is O(k + n) where k is the bucket size
+ */
+package com.gfred;
+public class RadixSort {
+
+    private static void radixSort(int[] array) {
+        protected int i;
```

Comment:
@gfredtech By reducing scope here I meant, use `i` directly in the loop itself.

Reply:
okay, I'm fixing that